app/models/user.rb

```diff
@@ -5,6 +5,8 @@ class User < ActiveRecord::Base
   USERNAME_CHARS  = "a-z0-9"
   USERNAME_FORMAT = /\A[#{USERNAME_CHARS}]{4,30}\Z/
 
+  APPLICATION_TOKENS_MAX = 5
```

Comment:
why not in the `ApplicationToken` model ? It looks like it would be more suitable there no ?